.utils/deploy.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# This file is used to deploy yttest to an S3 bucket. It
+# expects the S3 bucket name in an environment variable
+# $YTTEST_BUCKET
+
+set -ex
+
+main() {
+  # For short-lived assets; in seconds
+  FIVE_DAYS="432000"
+
+  # For long-lived assets; in seconds
+  ONE_YEAR="31536000"
+
+  CSPSTATIC="\"content-security-policy\": \"default-src 'none'; "\
+"base-uri 'none'; "\
+"form-action 'none'; "\
+"object-src 'none'\""
+  # CSP mimicing what https://ytlr-cert.appspot.com/2019/main.html uses
+  CSP="\"content-security-policy\": \"default-src * 'unsafe-inline' 'unsafe-eval' data: blob:;\""
```

Comment:
To put it mildly this is not so great. We're basically getting no XSS or clickjacking protection here. 

What is this CSP policy being used for? We should be able to make it less permissive.

Reply:
@whimboo does your team have any requirement for the CSP header for this web site?

@g-k also, what would secops team want us to use?

From ops perspective, we can use whatever CSP headers you guys want. The current CSP header mimics what https://ytlr-cert.appspot.com/2019/main.html uses, and this is meant to be reviewed, since we (ops) don't know what the requirement is for this site, as far as CSP header is concerned.

Reply:
I actually don't really know about the CSP headers. So I would defer to @g-k here. As long as the tests are working it's all fine.